modules/hal_nordic/nrfs/dvfs/Kconfig

```diff
@@ -30,6 +30,8 @@ config NRFS_LOCAL_DOMAIN_DOWNSCALE_FINISH_DELAY_TIMEOUT_US
 	int "Additional delay to let secdom finish dowscale procedure in us"
 	range 1 10000000
 	default 1000
+	help
+	  This value depends on the secdom core performance and shouldn't be touched by the user.
```

Comment:
Then why it is presented as a user-configurable Kconfig option?

Reply:
After talk with @anangl He advised to no change it to constant (not user configurable) since if user want's to break something he will do it, and with configurable option we have more flexibility and it is not hidden somewhere in code. @hubertmis is it ok for You ?